samples/java-eventsourced-counter/src/main/java/com/example/actions/CounterJournalToTopicActionTestKit.java

```diff
@@ -1,24 +1,25 @@
-package com.example.domain;
+/* This code is managed by Akka Serverless tooling.
+ * It will be re-generated to reflect any changes to your protobuf definitions.
+ * DO NOT EDIT
+ */
+
+package com.example.actions;
```

Comment:
Shouldn't be checked in now anymore that we have the codegen?

Reply:
It was added because of that, yes. Easier and less error-prone. 

You're right I can remove it now. I guess I was sentimentally attached to it :))